screenpy_selenium/common.py

```diff
@@ -0,0 +1,45 @@
+"""module to hold shared objects."""
```

Comment:
Should start with a capital letter. Shouldn't something catch this?

Reply:
I agree.  I would have thought the `D` rules should have caught this.

Reply:
Interesting. Even pure flake8-pydocstyle doesn't trigger it.